example/stochastic_subgrid/parthinput.stochastic_subgrid

```diff
@@ -0,0 +1,91 @@
+# ========================================================================================
+#  Athena++ astrophysical MHD code
+#  Copyright(C) 2014 James M. Stone <jmstone@princeton.edu> and other code contributors
+#  Licensed under the 3-clause BSD License, see LICENSE file for details
+# ========================================================================================
+#  (C) (or copyright) 2020-2021. Triad National Security, LLC. All rights reserved.
+#
+#  This program was produced under U.S. Government contract 89233218CNA000001 for Los
+#  Alamos National Laboratory (LANL), which is operated by Triad National Security, LLC
+#  for the U.S. Department of Energy/National Nuclear Security Administration. All rights
+#  in the program are reserved by Triad National Security, LLC, and the U.S. Department
+#  of Energy/National Nuclear Security Administration. The Government is granted for
+#  itself and others acting on its behalf a nonexclusive, paid-up, irrevocable worldwide
+#  license in this material to reproduce, prepare derivative works, distribute copies to
+#  the public, perform publicly and display publicly, and to permit others to do so.
+# ========================================================================================
+
+<parthenon/job>
+problem_id = advection
+
+<parthenon/mesh>
+refinement = adaptive
+numlevel = 3
+
+nx1 = 64
+x1min = -0.5
+x1max = 0.5
+ix1_bc = periodic
+ox1_bc = periodic
+
+nx2 = 64
+x2min = -0.5
+x2max = 0.5
+ix2_bc = periodic
+ox2_bc = periodic
+
+nx3 = 1
+x3min = -0.5
+x3max = 0.5
+ix3_bc = periodic
+ox3_bc = periodic
+
+<parthenon/meshblock>
+nx1 = 16
+nx2 = 16
+nx3 = 1
+
+<parthenon/time>
+nlim = -1
+tlim = 1.0
+integrator = rk2
+
+<Advection>
+cfl = 0.45
```

Comment:
What is cfl? Could we use something a little more descriptive?

Reply:
CFL is the Courant-Friedrichs-Lewy condition. This is extremely common and widely used in hydrodynamics and computational fluid dynamics. Although I'm usually always for more descriptive variable names, because "cfl" is used so ubiquitously, I think it would actually be more confusing to name it something different, because people expect it to be called "cfl".

Reply:
Ok, I guess my ignorance of this knowledge domain is showing. :)

Reply:
It's a fair question, though. I'm sure there are some variables that could be more descriptive.